src/lib/components/Controls.svelte

```diff
@@ -0,0 +1,174 @@
+<script>
+    import { times, duration } from "$lib/stores";
+	import { formatDuration, getCurrentTime } from "$lib/utils/time";
```

Comment:
Why's the formatting going funny?

Need to get a pre-commit linter running automatically.